mkdocs_swan/notebook.html

```diff
@@ -21,8 +32,8 @@
             {{ page.meta.notebook_name }} 
             
             {% if page.meta.notebook_url %}
-            <a href="{{ config.open_in_swan_url }}{{  config.gallery_url + page.meta.notebook_url | urlencode }}" title="Open in SWAN" class="md-content__button md-icon">
```

Comment:
Can we keep the url and just intercept the `onclick` event? I might want to chose to open in a new tab (ctrl/cmd+click).
Or it wouldn't work anyway?

Reply:
By keeping the url, it opens the session in a new SWAN instance, which is not the wanted behavior I assume? So, by intercepting the request with a `onclick` event I think it would be less practical.

Reply:
if you return false in the `onclick` handler, doesn't that prevent it?

Reply:
I tried at the beginning to have both the link and the `onclick` function but it prompts both of them at once. I assume that it's a normal behavior but I'm not sure if there was a work-around it.

Reply:
This works:
```html
<a href="https://cern.ch" onclick="myfunc(event)" target="_blank">Click me</a>
```
```js
myfunc = (e) => e.preventDefault()
```